packages/eas-cli/src/commands/submit.ts

```diff
@@ -174,4 +184,23 @@ export default class Submit extends EasCommand {
       nonInteractive,
     };
   }
+
+  private static sanitizePlatform(
```

Comment:
If you don't care about the result of a function like this, it should be named `validate...`:
```suggestion
  private static validatePlatform(
```

Reply:
```
// validate only
validateXyz(xyz);

// sanitize
const xyz = sanitizeXyz(rawXyz);
```

Reply:
Also, a nit - if I was writing this I'd drop `static`. I think there isn't any benefit from the keyword here. The method is only used in non-static methods.